css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  display: flex;
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-between;
+  max-height: 100vh;
+}
+
+.row {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-evenly;
+  flex-basis: auto;
+}
+
+.column-one-third {
+  width: calc(100% / 3);
+  flex-basis: 33%;
+}
+
```

Comment:
For the layout class we want to be using `flex-basis` and not `width`. Remove the width and replace with your `calc` function for your `flex-basis`